[2025-05-12T07:53:07.442+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_brewery.extract_breweries manual__2025-05-12T07:53:06.170638+00:00 [queued]>
[2025-05-12T07:53:07.446+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_brewery.extract_breweries manual__2025-05-12T07:53:06.170638+00:00 [queued]>
[2025-05-12T07:53:07.446+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-12T07:53:07.451+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): extract_breweries> on 2025-05-12 07:53:06.170638+00:00
[2025-05-12T07:53:07.453+0000] {standard_task_runner.py:57} INFO - Started process 1664 to run task
[2025-05-12T07:53:07.455+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_brewery', 'extract_breweries', 'manual__2025-05-12T07:53:06.170638+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/brewery_pipeline.py', '--cfg-path', '/tmp/tmp20a5gk9u']
[2025-05-12T07:53:07.456+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask extract_breweries
[2025-05-12T07:53:07.486+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_brewery.extract_breweries manual__2025-05-12T07:53:06.170638+00:00 [running]> on host 0d8312ffa556
[2025-05-12T07:53:07.522+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_brewery' AIRFLOW_CTX_TASK_ID='extract_breweries' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T07:53:06.170638+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T07:53:06.170638+00:00'
[2025-05-12T07:53:07.523+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-12T07:53:07.523+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/scripts/extract_breweries.py']
[2025-05-12T07:53:07.528+0000] {subprocess.py:86} INFO - Output:
[2025-05-12T07:53:07.730+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-05-12T07:53:07.730+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/extract_breweries.py", line 27, in <module>
[2025-05-12T07:53:07.733+0000] {subprocess.py:93} INFO -     run()
[2025-05-12T07:53:07.734+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/extract_breweries.py", line 19, in run
[2025-05-12T07:53:07.734+0000] {subprocess.py:93} INFO -     with open(filename, "w") as f:
[2025-05-12T07:53:07.734+0000] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: 'opt/***/data_lake/bronze/breweries_12052025_075307.json'
[2025-05-12T07:53:07.744+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-05-12T07:53:07.753+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-05-12T07:53:07.756+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=etl_brewery, task_id=extract_breweries, execution_date=20250512T075306, start_date=20250512T075307, end_date=20250512T075307
[2025-05-12T07:53:07.764+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 8 for task extract_breweries (Bash command failed. The command returned a non-zero exit code 1.; 1664)
[2025-05-12T07:53:07.770+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-12T07:53:07.782+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
